ironfish/src/wallet/walletdb/accountValue.ts

```diff
@@ -26,6 +27,7 @@ export interface AccountValue {
 export type AccountImport = Omit<AccountValue, 'id' | 'createdAt' | 'name'> & {
   createdAt: { hash: string; sequence: number } | null
   name?: string
+  language?: LanguageKey
```

Comment:
It feels a little odd to have `language` be part of the account

Is this to keep the `encode` method signature the same for `MnemonicEncoder`? I'm wondering if we should add `AccountEncodingOptions` or something for that

Reply:
I agree that makes sense to me